src/main/cpp/DragonVision/LimelightFactory.cpp

```diff
@@ -61,25 +61,26 @@ DragonLimelight *LimelightFactory::CreateLimelight(
     if (usage == LimelightUsages::PRIMARY)
     {
         if (m_limelight == nullptr)
-        {
-            m_limelight = new DragonLimelight(tableName,
-                                              mountingHeight,
-                                              mountingHorizontalOffset,
-                                              forwardOffset,
-                                              pitch,
-                                              yaw,
-                                              roll,
-                                              targetHeight,
-                                              targetHeight2,
-                                              ledMode,
-                                              camMode,
-                                              streamMode,
-                                              snapMode);
+        { /*
+             m_limelight = new DragonLimelight(tableName,
+                                               mountingHeight,
+                                               mountingHorizontalOffset,
+                                               forwardOffset,
+                                               pitch,
+                                               yaw,
+                                               roll,
+                                               targetHeight,
+                                               targetHeight2,
+                                               ledMode,
+                                               camMode,
+                                               streamMode,
+                                               snapMode);*/
         }
         return m_limelight;
     }
     else
```

Comment:
Factories are probably obsolete ... submit a task for  the code generator team to create your cameras